yarn.lock

```diff
@@ -1811,18 +1811,10 @@
     "@babel/helper-validator-identifier" "^7.24.7"
     to-fast-properties "^2.0.0"
 
-"@babel/types@^7.20.7", "@babel/types@^7.25.9", "@babel/types@^7.26.7", "@babel/types@^7.26.8":
-  version "7.26.8"
+"@babel/types@^7.25.9", "@babel/types@^7.26.7", "@babel/types@^7.26.8", "@babel/types@^7.27.1":
+  version "7.27.1"
```

Comment:
moved `v7.20.7` to the group above because it's has a dependency undeclared in this group, see [here](https://www.npmjs.com/package/@babel/types/v/7.20.7)

Reply:
Pushed d06921c17dd24dac32918ae9a4b97752cab7c597

Reply:
Apologies. That broke some types. Hopefully this fixes it 6ae7ef811bb631cc42d5f8c32f43adc3bae3aab2